app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java

```diff
@@ -504,4 +505,16 @@ private void getRuntimePermissions() {
             mPermissionGranted = true;
         }
     }
+
+    private boolean isPermissionGranted() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            if ((ContextCompat.checkSelfPermission(mActivity, Manifest.permission.WRITE_EXTERNAL_STORAGE)
+                    != PackageManager.PERMISSION_GRANTED) &&
+                    (ContextCompat.checkSelfPermission(mActivity, Manifest.permission.READ_EXTERNAL_STORAGE)
+                            != PackageManager.PERMISSION_GRANTED)) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
I would suggest you put this PR on hold since there is another PR which is adding a permission utils file in the project. After that PR is merged, you can use that utility class to check for permissions.

Reply:
that PR is still not approved by anyone. that's why I am saying to merge it first and then jddeep can fetch changes from main and use pemissionUtils function here also.
or maybe you can do the opposite, however you like.

Reply:
that PR has been merged. You can do the required changes now

Reply:
sure

Reply:
@Swati4star

Reply:
@Swati4star

Reply:
@Swati4star 
PermissionUtils class contains no method through which we can check if permission is already granted or not.(required in this case)
and what is the point of **checkRuntimePermissions** method returning true? it will return true only when `Build.VERSION.SDK_INT < Build.VERSION_CODES.M`
further ZipToPdfFragment still contains permission related code.
I don't understand why that PR was merged

Reply:
@Ni3verma You can create an issue regarding that. For your second query: https://github.com/Swati4star/Images-to-PDF/pull/656#discussion_r269386181

Reply:
@Swati4star please merge it as permissionUtils class is of no use here. and make a separate issue to fix permissionUtils and use it in ZipToPdfFragment and TextToPdfFragment